### PR TITLE
Add checkbox for en/dis wards of type. Small refactor

### LINKED ISSUE #1660: Make an additional checkbox for enables/disables all observer/sentry wards in matches/../vision
In the Vision section of a match [1], it would be nice to have an additional checkbox that enables/disables all the observer/sentry wards in a row[2], just like the checkboxes for Radiant/Dire do already [3].

[1] dotabuff-feature-1

[2] dotabuff-feature-3

[3] dotabuff-feature-2

### PR DESCRIPTION
**Pull request by S4elkyn4eGGG**
fixes #1660 

Add checkbox for enables/disables all observer/sentry wards in matches. Edit check all wards of team if at least one ward was checked. Small refactor.

### CODE REVIEW

**Comment by S4elkyn4eGGG** [comment]
for issue #1660

File: src/components/Match/Vision/VisionFilter.jsx [hunk 1]
```diff
@@ -51,6 +55,16 @@ class VisionFilter extends React.Component {
       this.playerColumn(2 + index),
       this.playerColumn(3 + index),
       this.playerColumn(4 + index),
+      {
+        displayName: 'All',
```
[the remark this answers is not in the document]
Comment (howardchung) [reply src/components/Match/Vision/VisionFilter.jsx]: I'm fine with the string, but agree it needs to be put into the `en-US.json` file.

[the remark this answers is not in the document]
Comment (S4elkyn4eGGG) [reply src/components/Match/Vision/VisionFilter.jsx]: I change word 'ALL' on chat_filter_all from en-US.json. Decided one more entry 'ALL' would be superfluous

File: src/lang/en-US.json [hunk 2]
```diff
@@ -64,7 +64,6 @@
   "api_month": "Month",
   "api_header_key": "Your Key",
   "api_header_table": "Get started for free. Keep going at a ridiculously low price.",
-  "api_header_details": "Details",
```
Comment (howardchung) [inline src/lang/en-US.json]: why is this line removed?

File: src/components/Match/Vision/VisionFilter.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (howardchung) [reply src/components/Match/Vision/VisionFilter.jsx]: yeah, we should reuse strings whenever possible :)

File: src/lang/en-US.json (hunk 2 above)
Comment (Argn0) [reply src/lang/en-US.json]: that line is a duplicate. line 49 and line 67

Comment (howardchung) [reply src/lang/en-US.json]: thanks, maybe we should add a build step to check for duplicates
